wsidicom/config.py

```diff
@@ -63,6 +64,17 @@ def focal_plane_distance_threshold(self) -> float:
     def focal_plane_distance_threshold(self, value: float) -> None:
         self._focal_plane_distance_threshold = value
 
+    @property
+    def pyramids_origin_threshold(self) -> float:
+        """Threshold in mm for the distance between origins of instances
+        to group them into the same pyramid. Default is 1 mm.
```

Comment:
The default looks to be set at 0.1 mm? I think this is a bit to much. The test image has a maximum of 0.0158 mm, so maybe 0.02 mm?